src/test/steps/generateReportSteps.ts

```diff
@@ -0,0 +1,50 @@
+import { Given, When, Then } from '@cucumber/cucumber';
+import { expect } from '@playwright/test';
+import basePage from '../../pages/basePage';
+
+When('User navigates to Reports tab', async function () {
```

Comment:
One step definition should be responsible for navigating user to ANY given page.